app/views/confirmation_page.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import views.html.Layout
+@import uk.gov.hmrc.auth.core.retrieve.Email
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukPanel
+
+@this(
+        layout: Layout,
+        link: components.link,
+        p: components.p,
+        h1: components.h1,
+        govukPanel: GovukPanel
+)
+
+@(
+    dates: String
+)(
+    implicit request: Request[_],
+    messages: Messages,
+    appConfig: AppConfig
+)
+
+@layout(
+    pageTitle = Some(messages("cf.cash-account.transactions.confirmation.statements")),
+    backLink = Some(controllers.routes.SelectedTransactionsController.onPageLoad().url),
+    fullWidth = false,
+    help = false
+) {
+
+    @govukPanel(Panel(
+        title = Text(messages("cf.cash-account.transactions.confirmation.statements")),
+        headingLevel = 1,
+        content = Text(dates)
+    ))
+
+    @p(
+        message = messages(s"cf.cash-account.transactions.confirmation.next"),
+        classes = "govuk-heading-s",
+        id=Some("email-confirmation-subheader")
+    )
+
+    @p(
+        message = s"${messages("cf.cash-account.transactions.confirmation.email")}",
+        classes = "govuk-body",
```

Comment:
we don't need to apply `govuk-body` since its applied by default on the main component anyway